### ISSUE TITLE
Videos no longer being added to history

### ISSUE THREAD

**Issue by Nethanswarm** [yuliskov/SmartTubeNext#2123]
Last night, it was working fine, and then suddenly, if I watched a video, it would no longer be added to my history.

I use the app on a firestick. I tried clearing the cache, clearing the app data, restarting the firestick and uninstalling and then reinstalling the app, still doesn't add videos to my history. Definitely logged in (had to relog in after clearing app data and reinstalling).

On my computer and phone, even starting watching a video adds the video to my history, even doing so on the normal Youtube app on the same firestick does so.

Anyone else got this problem? And any solutions possible?

**Comment by perariontaeadastra** [comment]
Same here. Also starting yesterday. Sony Bravia tv. Tried same solutions as the topic starter. Nothing helped. History is working fine on other devices.

**Comment by Nethanswarm** [comment]
Ah I installed the beta version and saw there was an update which included a fix to the problem. Videos are being added to history again, thanks!

**Comment by DrJaymz** [comment]
Yes, the current version has this problem again. It makes following multiple series video all but impossible. Hopefully the fix will find its was to the release stream asap.

**Comment by falcorr** [comment]
I don't think it's ever added videos to history. At least for a very long time.
